webpack.config.test.js

```diff
@@ -1,13 +1,16 @@
+/** Used in .babelrc for 'test' environment */
+
 // for babel-plugin-webpack-loaders
 require('babel-register');
-const devConfigs = require('./webpack.config.development');
 const validate = require('webpack-validator');
+const devConfig = require('./webpack.config.development');
 
 module.exports = validate({
   output: {
     libraryTarget: 'commonjs2'
   },
   module: {
-    loaders: devConfigs.module.loaders.slice(1)  // remove babel-loader
+    // Use base + development loaders, but exclude 'babel-loader'
+    loaders: devConfig.module.loaders.slice(1)
```

Comment:
It seems to me that `devConfig.module.loaders` does not contain babel-loader … did that change or do I not see something?